src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -403,6 +403,11 @@ MTR_EXTERN NSString * const MTRDataVersionKey MTR_NEWLY_AVAILABLE;
  */
 - (void)deviceBecameActive:(MTRDevice *)device MTR_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4));
 
+/**
+ * Notifies delegate when the device attribute cache has been primed with initial configuration data of the device
```

Comment:
I think this needs to more clearly explain what "configuration data" is.  I was expecting this to happen at the end of the subscription priming read, but this can happen any time we have all the descriptor clusters, even if we don't have the other bits yet....

And the fact that _usually_ this will happen at the end of the priming read (if the delegate is added early) but then at a different time if the delegate is added later is a bit odd.

Reply:
Note, this will only be primed once, ever, after the first subscription post pairing, if the client implements storage. @jtung-apple is working on making that change. This isn't expected to be called more than that.

Reply:
The change to make this call back only once is done in the latest commit.